elements/classpower.lua

```diff
@@ -144,9 +144,10 @@ local function Update(self, event, unit, powerType)
 	* max           - the maximum amount of power (number)
 	* hasMaxChanged - indicates whether the maximum amount has changed since the last update (boolean)
 	* powerType     - the active power type (string)
+	* isVisible     - whether the element is visible (boolean?)
```

Comment:
Out of curiosity, don't you just want to call this `isEnabled` to match the internal property?

Reply:
I'm not sure if it would cause confusion with [:IsElementEnabled](https://github.com/oUF-wow/oUF/blob/13f2f6db9109b5025c5b6c89f2fdb6bbb6dddf8b/ouf.lua#L147-L161) which is something different.

Reply:
Then the variable should match imo.

Reply:
I mean, it's your codebase of course, I'm just a bystander, so it's all up to you guys - but since this is essentially already a thing, introducing a third variable might cause more confusion rather than reduce it?

Reply:
Yes `.__isVisible` is a better name I suppose

Reply:
Hmm, isn't `.isEnabled` used in other elements as well? I guess if you want to change it, it would be good to update it across the board?

Reply:
[No, it isn't](https://github.com/oUF-wow/oUF/search?q=isEnabled&unscoped_q=isEnabled)

Reply:
Fair enough! My bad. :)